hxrsnd/bragg.py

```diff
@@ -302,7 +302,7 @@ def d_space(ID, hkl):
     ID = check_id(ID)
     h = hkl[0]
     k = hkl[1]
-    l = hkl[2]
+    l_ = hkl[2]
```

Comment:
Out of curiosity, what was the problem here?

Reply:
"Ambiguous variable name" `l` looks like `1`